drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -173,12 +173,14 @@ void RasterizerGLES2::initialize() {
 #endif // GLAD_ENABLED
 
 	// For debugging
+#if 0
```

Comment:
Hmmmmmmmm well the debugging is quite useful when it's available, just using a `#if 0` seems a bit radical. 

Couldn't the `GLES_OVER_GL` flag be used to find out if this is supported or not or something?